code/src/components/Album.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import { CoverArt } from 'components/CoverArt';
+import { Title } from 'components/Title';
+import { Artists } from 'components/Artists';
+
+export const Album = (props) => {
+  return (
+    <div className="body">
+      <CoverArt link={props.item.external_urls.spotify} images={props.item.images} />
```

Comment:
Might be something in the name of this reference when you are maping external urls that is not as it should?